playbooks/common/openshift-master/config.yml

```diff
@@ -337,12 +361,54 @@
     with_items: openshift_master_named_certificates
     when: named_certs_specified | bool
 
-- name: Configure master instances
-  hosts: oo_masters_to_config
+- name: Configure first master instance
```

Comment:
Hmm, although, if I calculated *precise* openshift_version for first master in the playbook, perhaps just by running the openshift_docker role and not much else, I could then set that on all masters, and then run configure on all masters just once. We'd be down to just one block.